src/pyfaf/opsys/Makefile.am

```diff
@@ -4,3 +4,15 @@ opsys_PYTHON = \
     fedora.py
 
 opsysdir = $(pythondir)/pyfaf/opsys
+
+EXTRA_DIST = centos.py.in
+
+.PHONY: centos.py
+centos.py: centos.py.in
+if USE_DNF
+	sed -e s,\@Sw_mgr\@,Dnf,g -e s,\@sw_mgr\@,dnf,g $< >$@
+endif
+if USE_YUM
+	sed -e s,\@Sw_mgr\@,Yum,g -e s,\@sw_mgr\@,yum,g $< >$@
+endif
+	chmod +x $@
```

Comment:
I am not sure about this. I would say the executable is not needed here.

Reply:
You are right. It will be updated in my next commit.